webapp/src/Controller/Jury/AnalysisController.php

```diff
@@ -99,6 +100,7 @@ public function teamAction(Team $team): Response
 
     #[Route(path: '/problem/{probid}', name: 'analysis_problem')]
     public function problemAction(
+        #[MapEntity(id: 'probid')]
```

Comment:
When do we use this?

Reply:
Map entity makes it so that `$problem` gets injected properly using the `probid` route argument as primary key (i.e. as `id`, not a field called `id`) in the method. This used to be automatic behavior but with `doctrine.orm.controller_resolver.auto_mapping: false`, which will be the default soon, this is not the case anymore